app/main.py

```diff
@@ -2,0 +1,10 @@
+def calculate_team_total_rating(player: list) -> int:
+    return sum([player.get_rating() for player in player])
+
+
+def elves_concert(elf: list) -> None:
+    [el.play_elf_song() for el in elf]
```

Comment:
The list comprehension `[el.play_elf_song() for el in elf]` is unnecessary since the result is not being used. You can use a simple for loop instead: `for el in elf: el.play_elf_song()`. This will prevent creating a redundant list and improve code efficiency .